2023-01-09T04:53:07.568082761Z I0109 04:53:07.567951       1 flags.go:64] FLAG: --accesstoken-inactivity-timeout="0s"
2023-01-09T04:53:07.568082761Z I0109 04:53:07.568057       1 flags.go:64] FLAG: --add-dir-header="false"
2023-01-09T04:53:07.568082761Z I0109 04:53:07.568065       1 flags.go:64] FLAG: --admission-control-config-file=""
2023-01-09T04:53:07.568165559Z I0109 04:53:07.568075       1 flags.go:64] FLAG: --advertise-address="<nil>"
2023-01-09T04:53:07.568165559Z I0109 04:53:07.568084       1 flags.go:64] FLAG: --alsologtostderr="false"
2023-01-09T04:53:07.568165559Z I0109 04:53:07.568092       1 flags.go:64] FLAG: --api-audiences="[https://kubernetes.default.svc]"
2023-01-09T04:53:07.568165559Z I0109 04:53:07.568108       1 flags.go:64] FLAG: --audit-log-batch-buffer-size="10000"
2023-01-09T04:53:07.568165559Z I0109 04:53:07.568118       1 flags.go:64] FLAG: --audit-log-batch-max-size="1"
2023-01-09T04:53:07.568165559Z I0109 04:53:07.568125       1 flags.go:64] FLAG: --audit-log-batch-max-wait="0s"
2023-01-09T04:53:07.568165559Z I0109 04:53:07.568132       1 flags.go:64] FLAG: --audit-log-batch-throttle-burst="0"
2023-01-09T04:53:07.568165559Z I0109 04:53:07.568139       1 flags.go:64] FLAG: --audit-log-batch-throttle-enable="false"
2023-01-09T04:53:07.568165559Z I0109 04:53:07.568146       1 flags.go:64] FLAG: --audit-log-batch-throttle-qps="0"
2023-01-09T04:53:07.568165559Z I0109 04:53:07.568154       1 flags.go:64] FLAG: --audit-log-compress="false"
2023-01-09T04:53:07.568176462Z I0109 04:53:07.568162       1 flags.go:64] FLAG: --audit-log-format="json"
2023-01-09T04:53:07.568176462Z I0109 04:53:07.568169       1 flags.go:64] FLAG: --audit-log-maxage="0"
2023-01-09T04:53:07.568184276Z I0109 04:53:07.568175       1 flags.go:64] FLAG: --audit-log-maxbackup="10"
2023-01-09T04:53:07.568191540Z I0109 04:53:07.568182       1 flags.go:64] FLAG: --audit-log-maxsize="100"
2023-01-09T04:53:07.568198783Z I0109 04:53:07.568189       1 flags.go:64] FLAG: --audit-log-mode="blocking"
2023-01-09T04:53:07.568205951Z I0109 04:53:07.568196       1 flags.go:64] FLAG: --audit-log-path="/var/log/oauth-apiserver/audit.log"
2023-01-09T04:53:07.568213189Z I0109 04:53:07.568203       1 flags.go:64] FLAG: --audit-log-truncate-enabled="false"
2023-01-09T04:53:07.568220382Z I0109 04:53:07.568210       1 flags.go:64] FLAG: --audit-log-truncate-max-batch-size="10485760"
2023-01-09T04:53:07.568227662Z I0109 04:53:07.568219       1 flags.go:64] FLAG: --audit-log-truncate-max-event-size="102400"
2023-01-09T04:53:07.568234909Z I0109 04:53:07.568226       1 flags.go:64] FLAG: --audit-log-version="audit.k8s.io/v1"
2023-01-09T04:53:07.568242208Z I0109 04:53:07.568233       1 flags.go:64] FLAG: --audit-policy-file="/var/run/configmaps/audit/policy.yaml"
2023-01-09T04:53:07.568249455Z I0109 04:53:07.568240       1 flags.go:64] FLAG: --audit-webhook-batch-buffer-size="10000"
2023-01-09T04:53:07.568256627Z I0109 04:53:07.568248       1 flags.go:64] FLAG: --audit-webhook-batch-initial-backoff="10s"
2023-01-09T04:53:07.568275461Z I0109 04:53:07.568261       1 flags.go:64] FLAG: --audit-webhook-batch-max-size="400"
2023-01-09T04:53:07.568283906Z I0109 04:53:07.568273       1 flags.go:64] FLAG: --audit-webhook-batch-max-wait="30s"
2023-01-09T04:53:07.568283906Z I0109 04:53:07.568280       1 flags.go:64] FLAG: --audit-webhook-batch-throttle-burst="15"
2023-01-09T04:53:07.568293059Z I0109 04:53:07.568286       1 flags.go:64] FLAG: --audit-webhook-batch-throttle-enable="true"
2023-01-09T04:53:07.568301962Z I0109 04:53:07.568293       1 flags.go:64] FLAG: --audit-webhook-batch-throttle-qps="10"
2023-01-09T04:53:07.568309139Z I0109 04:53:07.568300       1 flags.go:64] FLAG: --audit-webhook-config-file=""
2023-01-09T04:53:07.568320143Z I0109 04:53:07.568307       1 flags.go:64] FLAG: --audit-webhook-initial-backoff="10s"
2023-01-09T04:53:07.568320143Z I0109 04:53:07.568314       1 flags.go:64] FLAG: --audit-webhook-mode="batch"
2023-01-09T04:53:07.568327722Z I0109 04:53:07.568321       1 flags.go:64] FLAG: --audit-webhook-truncate-enabled="false"
2023-01-09T04:53:07.568334899Z I0109 04:53:07.568327       1 flags.go:64] FLAG: --audit-webhook-truncate-max-batch-size="10485760"
2023-01-09T04:53:07.568342082Z I0109 04:53:07.568334       1 flags.go:64] FLAG: --audit-webhook-truncate-max-event-size="102400"
2023-01-09T04:53:07.568349282Z I0109 04:53:07.568341       1 flags.go:64] FLAG: --audit-webhook-version="audit.k8s.io/v1"
2023-01-09T04:53:07.568349282Z I0109 04:53:07.568346       1 flags.go:64] FLAG: --authentication-kubeconfig=""
2023-01-09T04:53:07.568356694Z I0109 04:53:07.568350       1 flags.go:64] FLAG: --authentication-skip-lookup="false"
2023-01-09T04:53:07.568363906Z I0109 04:53:07.568355       1 flags.go:64] FLAG: --authentication-token-webhook-cache-ttl="10s"
2023-01-09T04:53:07.568363906Z I0109 04:53:07.568360       1 flags.go:64] FLAG: --authentication-tolerate-lookup-failure="false"
2023-01-09T04:53:07.568382382Z I0109 04:53:07.568365       1 flags.go:64] FLAG: --authorization-always-allow-paths="[/healthz,/readyz,/livez]"
2023-01-09T04:53:07.568390630Z I0109 04:53:07.568380       1 flags.go:64] FLAG: --authorization-kubeconfig=""
2023-01-09T04:53:07.568390630Z I0109 04:53:07.568385       1 flags.go:64] FLAG: --authorization-webhook-cache-authorized-ttl="10s"
2023-01-09T04:53:07.568398007Z I0109 04:53:07.568390       1 flags.go:64] FLAG: --authorization-webhook-cache-unauthorized-ttl="10s"
2023-01-09T04:53:07.568405174Z I0109 04:53:07.568398       1 flags.go:64] FLAG: --bind-address="0.0.0.0"
2023-01-09T04:53:07.568412313Z I0109 04:53:07.568403       1 flags.go:64] FLAG: --cert-dir="apiserver.local.config/certificates"
2023-01-09T04:53:07.568412313Z I0109 04:53:07.568409       1 flags.go:64] FLAG: --client-ca-file=""
2023-01-09T04:53:07.568419692Z I0109 04:53:07.568414       1 flags.go:64] FLAG: --contention-profiling="false"
2023-01-09T04:53:07.568428471Z I0109 04:53:07.568418       1 flags.go:64] FLAG: --cors-allowed-origins="[//127\\.0\\.0\\.1(:|$),//localhost(:|$)]"
2023-01-09T04:53:07.568435780Z I0109 04:53:07.568428       1 flags.go:64] FLAG: --default-watch-cache-size="100"
2023-01-09T04:53:07.568442948Z I0109 04:53:07.568433       1 flags.go:64] FLAG: --delete-collection-workers="1"
2023-01-09T04:53:07.568450366Z I0109 04:53:07.568438       1 flags.go:64] FLAG: --disable-admission-plugins="[]"
2023-01-09T04:53:07.568457484Z I0109 04:53:07.568447       1 flags.go:64] FLAG: --egress-selector-config-file=""
2023-01-09T04:53:07.568464654Z I0109 04:53:07.568452       1 flags.go:64] FLAG: --enable-admission-plugins="[]"
2023-01-09T04:53:07.568464654Z I0109 04:53:07.568460       1 flags.go:64] FLAG: --enable-garbage-collector="true"
2023-01-09T04:53:07.568471992Z I0109 04:53:07.568465       1 flags.go:64] FLAG: --enable-priority-and-fairness="true"
2023-01-09T04:53:07.568479641Z I0109 04:53:07.568470       1 flags.go:64] FLAG: --encryption-provider-config=""
2023-01-09T04:53:07.568479641Z I0109 04:53:07.568475       1 flags.go:64] FLAG: --etcd-cafile="/var/run/configmaps/etcd-serving-ca/ca-bundle.crt"
2023-01-09T04:53:07.568488715Z I0109 04:53:07.568481       1 flags.go:64] FLAG: --etcd-certfile="/var/run/secrets/etcd-client/tls.crt"
2023-01-09T04:53:07.568495957Z I0109 04:53:07.568487       1 flags.go:64] FLAG: --etcd-compaction-interval="5m0s"
2023-01-09T04:53:07.568495957Z I0109 04:53:07.568492       1 flags.go:64] FLAG: --etcd-count-metric-poll-period="1m0s"
2023-01-09T04:53:07.568506697Z I0109 04:53:07.568497       1 flags.go:64] FLAG: --etcd-db-metric-poll-interval="30s"
2023-01-09T04:53:07.568506697Z I0109 04:53:07.568502       1 flags.go:64] FLAG: --etcd-healthcheck-timeout="10s"
2023-01-09T04:53:07.568519782Z I0109 04:53:07.568511       1 flags.go:64] FLAG: --etcd-keyfile="/var/run/secrets/etcd-client/tls.key"
2023-01-09T04:53:07.568527389Z I0109 04:53:07.568517       1 flags.go:64] FLAG: --etcd-prefix="openshift.io"
2023-01-09T04:53:07.568536216Z I0109 04:53:07.568522       1 flags.go:64] FLAG: --etcd-servers="[https://10.0.145.4:2379,https://10.0.160.211:2379,https://10.0.199.219:2379]"
2023-01-09T04:53:07.568545069Z I0109 04:53:07.568535       1 flags.go:64] FLAG: --etcd-servers-overrides="[]"
2023-01-09T04:53:07.568552246Z I0109 04:53:07.568543       1 flags.go:64] FLAG: --external-hostname=""
2023-01-09T04:53:07.568559448Z I0109 04:53:07.568548       1 flags.go:64] FLAG: --feature-gates=""
2023-01-09T04:53:07.568559448Z I0109 04:53:07.568555       1 flags.go:64] FLAG: --goaway-chance="0"
2023-01-09T04:53:07.568568433Z I0109 04:53:07.568561       1 flags.go:64] FLAG: --help="false"
2023-01-09T04:53:07.568575580Z I0109 04:53:07.568566       1 flags.go:64] FLAG: --http2-max-streams-per-connection="1000"
2023-01-09T04:53:07.568575580Z I0109 04:53:07.568571       1 flags.go:64] FLAG: --kubeconfig=""
2023-01-09T04:53:07.568582899Z I0109 04:53:07.568576       1 flags.go:64] FLAG: --lease-reuse-duration-seconds="60"
2023-01-09T04:53:07.568590038Z I0109 04:53:07.568581       1 flags.go:64] FLAG: --livez-grace-period="0s"
2023-01-09T04:53:07.568597149Z I0109 04:53:07.568585       1 flags.go:64] FLAG: --log-backtrace-at=":0"
2023-01-09T04:53:07.568597149Z I0109 04:53:07.568593       1 flags.go:64] FLAG: --log-dir=""
2023-01-09T04:53:07.568604482Z I0109 04:53:07.568598       1 flags.go:64] FLAG: --log-file=""
2023-01-09T04:53:07.568611668Z I0109 04:53:07.568603       1 flags.go:64] FLAG: --log-file-max-size="1800"
2023-01-09T04:53:07.568611668Z I0109 04:53:07.568608       1 flags.go:64] FLAG: --log-flush-frequency="5s"
2023-01-09T04:53:07.568619058Z I0109 04:53:07.568613       1 flags.go:64] FLAG: --logtostderr="true"
2023-01-09T04:53:07.568626210Z I0109 04:53:07.568617       1 flags.go:64] FLAG: --master-service-namespace="default"
2023-01-09T04:53:07.568633358Z I0109 04:53:07.568626       1 flags.go:64] FLAG: --max-mutating-requests-inflight="200"
2023-01-09T04:53:07.568640478Z I0109 04:53:07.568631       1 flags.go:64] FLAG: --max-requests-inflight="400"
2023-01-09T04:53:07.568640478Z I0109 04:53:07.568636       1 flags.go:64] FLAG: --min-request-timeout="1800"
2023-01-09T04:53:07.568647896Z I0109 04:53:07.568640       1 flags.go:64] FLAG: --one-output="false"
2023-01-09T04:53:07.568655060Z I0109 04:53:07.568645       1 flags.go:64] FLAG: --permit-address-sharing="false"
2023-01-09T04:53:07.568655060Z I0109 04:53:07.568650       1 flags.go:64] FLAG: --permit-port-sharing="false"
2023-01-09T04:53:07.568662536Z I0109 04:53:07.568655       1 flags.go:64] FLAG: --profiling="true"
2023-01-09T04:53:07.568669645Z I0109 04:53:07.568660       1 flags.go:64] FLAG: --request-timeout="1m0s"
2023-01-09T04:53:07.568676798Z I0109 04:53:07.568665       1 flags.go:64] FLAG: --requestheader-allowed-names="[]"
2023-01-09T04:53:07.568676798Z I0109 04:53:07.568673       1 flags.go:64] FLAG: --requestheader-client-ca-file=""
2023-01-09T04:53:07.568700703Z I0109 04:53:07.568678       1 flags.go:64] FLAG: --requestheader-extra-headers-prefix="[x-remote-extra-]"
2023-01-09T04:53:07.568700703Z I0109 04:53:07.568691       1 flags.go:64] FLAG: --requestheader-group-headers="[x-remote-group]"
2023-01-09T04:53:07.568713122Z I0109 04:53:07.568702       1 flags.go:64] FLAG: --requestheader-username-headers="[x-remote-user]"
2023-01-09T04:53:07.568720490Z I0109 04:53:07.568712       1 flags.go:64] FLAG: --secure-port="8443"
2023-01-09T04:53:07.568720490Z I0109 04:53:07.568717       1 flags.go:64] FLAG: --shutdown-delay-duration="15s"
2023-01-09T04:53:07.568727844Z I0109 04:53:07.568722       1 flags.go:64] FLAG: --shutdown-send-retry-after="false"
2023-01-09T04:53:07.568735003Z I0109 04:53:07.568726       1 flags.go:64] FLAG: --skip-headers="false"
2023-01-09T04:53:07.568735003Z I0109 04:53:07.568731       1 flags.go:64] FLAG: --skip-log-headers="false"
2023-01-09T04:53:07.568742347Z I0109 04:53:07.568736       1 flags.go:64] FLAG: --stderrthreshold="2"
2023-01-09T04:53:07.568751126Z I0109 04:53:07.568744       1 flags.go:64] FLAG: --storage-backend=""
2023-01-09T04:53:07.568758268Z I0109 04:53:07.568749       1 flags.go:64] FLAG: --storage-media-type="application/json"
2023-01-09T04:53:07.568765374Z I0109 04:53:07.568754       1 flags.go:64] FLAG: --strict-transport-security-directives="[]"
2023-01-09T04:53:07.568772565Z I0109 04:53:07.568763       1 flags.go:64] FLAG: --tls-cert-file="/var/run/secrets/serving-cert/tls.crt"
2023-01-09T04:53:07.568792977Z I0109 04:53:07.568769       1 flags.go:64] FLAG: --tls-cipher-suites="[TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256]"
2023-01-09T04:53:07.568819975Z I0109 04:53:07.568790       1 flags.go:64] FLAG: --tls-min-version="VersionTLS12"
2023-01-09T04:53:07.568819975Z I0109 04:53:07.568795       1 flags.go:64] FLAG: --tls-private-key-file="/var/run/secrets/serving-cert/tls.key"
2023-01-09T04:53:07.568830662Z I0109 04:53:07.568817       1 flags.go:64] FLAG: --tls-sni-cert-key="[]"
2023-01-09T04:53:07.568830662Z I0109 04:53:07.568825       1 flags.go:64] FLAG: --tracing-config-file=""
2023-01-09T04:53:07.568838367Z I0109 04:53:07.568830       1 flags.go:64] FLAG: --v="2"
2023-01-09T04:53:07.568838367Z I0109 04:53:07.568835       1 flags.go:64] FLAG: --vmodule=""
2023-01-09T04:53:07.568847477Z I0109 04:53:07.568840       1 flags.go:64] FLAG: --watch-cache="true"
2023-01-09T04:53:07.568868655Z I0109 04:53:07.568845       1 flags.go:64] FLAG: --watch-cache-sizes="[]"
2023-01-09T04:53:07.569958146Z I0109 04:53:07.569925       1 dynamic_serving_content.go:113] "Loaded a new cert/key pair" name="serving-cert::/var/run/secrets/serving-cert/tls.crt::/var/run/secrets/serving-cert/tls.key"
2023-01-09T04:53:07.983570446Z I0109 04:53:07.983254       1 requestheader_controller.go:244] Loaded a new request header values for RequestHeaderAuthRequestController
2023-01-09T04:53:07.984282149Z I0109 04:53:07.984243       1 audit.go:349] Using audit backend: ignoreErrors<log>
2023-01-09T04:53:07.984995048Z I0109 04:53:07.984973       1 plugins.go:158] Loaded 2 mutating admission controller(s) successfully in the following order: NamespaceLifecycle,MutatingAdmissionWebhook.
2023-01-09T04:53:07.984995048Z I0109 04:53:07.984989       1 plugins.go:161] Loaded 1 validating admission controller(s) successfully in the following order: ValidatingAdmissionWebhook.
2023-01-09T04:53:07.985148953Z I0109 04:53:07.985120       1 apf_controller.go:218] NewTestableController "Controller" with serverConcurrencyLimit=600, requestWaitLimit=15s, name=Controller, asFieldManager="api-priority-and-fairness-config-consumer-v1"
2023-01-09T04:53:08.008832520Z I0109 04:53:08.008771       1 store.go:1452] "Monitoring resource count at path" resource="oauthclients.oauth.openshift.io" path="<storage-prefix>//oauth/clients"
2023-01-09T04:53:08.011755220Z I0109 04:53:08.011718       1 cacher.go:410] cacher (*oauth.OAuthClient): initialized
2023-01-09T04:53:08.020831546Z I0109 04:53:08.020770       1 store.go:1452] "Monitoring resource count at path" resource="oauthauthorizetokens.oauth.openshift.io" path="<storage-prefix>//oauth/authorizetokens"
2023-01-09T04:53:08.022534170Z I0109 04:53:08.022465       1 cacher.go:410] cacher (*oauth.OAuthAuthorizeToken): initialized
2023-01-09T04:53:08.028599291Z I0109 04:53:08.028574       1 store.go:1452] "Monitoring resource count at path" resource="oauthaccesstokens.oauth.openshift.io" path="<storage-prefix>//oauth/accesstokens"
2023-01-09T04:53:08.030784113Z I0109 04:53:08.030760       1 cacher.go:410] cacher (*oauth.OAuthAccessToken): initialized
2023-01-09T04:53:08.038026961Z I0109 04:53:08.037998       1 store.go:1452] "Monitoring resource count at path" resource="oauthclientauthorizations.oauth.openshift.io" path="<storage-prefix>//oauth/clientauthorizations"
2023-01-09T04:53:08.039578151Z I0109 04:53:08.039554       1 cacher.go:410] cacher (*oauth.OAuthClientAuthorization): initialized
2023-01-09T04:53:08.070699090Z I0109 04:53:08.070665       1 store.go:1452] "Monitoring resource count at path" resource="users.user.openshift.io" path="<storage-prefix>//users"
2023-01-09T04:53:08.072513445Z I0109 04:53:08.072484       1 cacher.go:410] cacher (*user.User): initialized
2023-01-09T04:53:08.078969940Z I0109 04:53:08.078939       1 store.go:1452] "Monitoring resource count at path" resource="identities.user.openshift.io" path="<storage-prefix>//useridentities"
2023-01-09T04:53:08.080918360Z I0109 04:53:08.080870       1 cacher.go:410] cacher (*user.Identity): initialized
2023-01-09T04:53:08.086178349Z I0109 04:53:08.086152       1 store.go:1452] "Monitoring resource count at path" resource="groups.user.openshift.io" path="<storage-prefix>//groups"
2023-01-09T04:53:08.091262351Z I0109 04:53:08.091237       1 cacher.go:410] cacher (*user.Group): initialized
2023-01-09T04:53:08.315729250Z I0109 04:53:08.315680       1 aggregator.go:115] Building initial OpenAPI spec
2023-01-09T04:53:08.318458643Z I0109 04:53:08.318421       1 aggregator.go:118] Finished initial OpenAPI spec generation after 2.71702ms
2023-01-09T04:53:08.318639940Z I0109 04:53:08.318606       1 genericapiserver.go:418] MuxAndDiscoveryComplete has all endpoints registered and discovery information is complete
2023-01-09T04:53:08.323696097Z I0109 04:53:08.323666       1 requestheader_controller.go:169] Starting RequestHeaderAuthRequestController
2023-01-09T04:53:08.323696097Z I0109 04:53:08.323689       1 shared_informer.go:255] Waiting for caches to sync for RequestHeaderAuthRequestController
2023-01-09T04:53:08.323758707Z I0109 04:53:08.323735       1 configmap_cafile_content.go:202] "Starting controller" name="client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file"
2023-01-09T04:53:08.323773608Z I0109 04:53:08.323749       1 configmap_cafile_content.go:202] "Starting controller" name="client-ca::kube-system::extension-apiserver-authentication::client-ca-file"
2023-01-09T04:53:08.323773608Z I0109 04:53:08.323767       1 shared_informer.go:255] Waiting for caches to sync for client-ca::kube-system::extension-apiserver-authentication::client-ca-file
2023-01-09T04:53:08.323784143Z I0109 04:53:08.323755       1 shared_informer.go:255] Waiting for caches to sync for client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file
2023-01-09T04:53:08.324001866Z I0109 04:53:08.323976       1 dynamic_serving_content.go:132] "Starting controller" name="serving-cert::/var/run/secrets/serving-cert/tls.crt::/var/run/secrets/serving-cert/tls.key"
2023-01-09T04:53:08.324070076Z I0109 04:53:08.324044       1 tlsconfig.go:200] "Loaded serving cert" certName="serving-cert::/var/run/secrets/serving-cert/tls.crt::/var/run/secrets/serving-cert/tls.key" certDetail="\"api.openshift-oauth-apiserver.svc\" [serving] validServingFor=[api.openshift-oauth-apiserver.svc,api.openshift-oauth-apiserver.svc.cluster.local] issuer=\"openshift-service-serving-signer@1673239264\" (2023-01-09 04:41:24 +0000 UTC to 2025-01-08 04:41:25 +0000 UTC (now=2023-01-09 04:53:08.324002626 +0000 UTC))"
2023-01-09T04:53:08.324257623Z I0109 04:53:08.324233       1 named_certificates.go:53] "Loaded SNI cert" index=0 certName="self-signed loopback" certDetail="\"apiserver-loopback-client@1673239987\" [serving] validServingFor=[apiserver-loopback-client] issuer=\"apiserver-loopback-client-ca@1673239987\" (2023-01-09 03:53:07 +0000 UTC to 2024-01-09 03:53:07 +0000 UTC (now=2023-01-09 04:53:08.324210243 +0000 UTC))"
2023-01-09T04:53:08.324316213Z I0109 04:53:08.324299       1 secure_serving.go:210] Serving securely on [::]:8443
2023-01-09T04:53:08.324346000Z I0109 04:53:08.324328       1 genericapiserver.go:488] [graceful-termination] waiting for shutdown to be initiated
2023-01-09T04:53:08.324370385Z I0109 04:53:08.324339       1 tlsconfig.go:240] "Starting DynamicServingCertificateController"
2023-01-09T04:53:08.324417698Z I0109 04:53:08.324397       1 apf_controller.go:317] Starting API Priority and Fairness config controller
2023-01-09T04:53:08.425843840Z I0109 04:53:08.424695       1 shared_informer.go:262] Caches are synced for client-ca::kube-system::extension-apiserver-authentication::client-ca-file
2023-01-09T04:53:08.425843840Z I0109 04:53:08.424756       1 shared_informer.go:262] Caches are synced for client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file
2023-01-09T04:53:08.425843840Z I0109 04:53:08.425042       1 tlsconfig.go:178] "Loaded client CA" index=0 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"aggregator-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:22 +0000 UTC to 2023-01-10 04:28:22 +0000 UTC (now=2023-01-09 04:53:08.425013958 +0000 UTC))"
2023-01-09T04:53:08.425843840Z I0109 04:53:08.425146       1 apf_controller.go:322] Running API Priority and Fairness config worker
2023-01-09T04:53:08.425843840Z I0109 04:53:08.425308       1 tlsconfig.go:200] "Loaded serving cert" certName="serving-cert::/var/run/secrets/serving-cert/tls.crt::/var/run/secrets/serving-cert/tls.key" certDetail="\"api.openshift-oauth-apiserver.svc\" [serving] validServingFor=[api.openshift-oauth-apiserver.svc,api.openshift-oauth-apiserver.svc.cluster.local] issuer=\"openshift-service-serving-signer@1673239264\" (2023-01-09 04:41:24 +0000 UTC to 2025-01-08 04:41:25 +0000 UTC (now=2023-01-09 04:53:08.425269869 +0000 UTC))"
2023-01-09T04:53:08.425843840Z I0109 04:53:08.425480       1 named_certificates.go:53] "Loaded SNI cert" index=0 certName="self-signed loopback" certDetail="\"apiserver-loopback-client@1673239987\" [serving] validServingFor=[apiserver-loopback-client] issuer=\"apiserver-loopback-client-ca@1673239987\" (2023-01-09 03:53:07 +0000 UTC to 2024-01-09 03:53:07 +0000 UTC (now=2023-01-09 04:53:08.425459929 +0000 UTC))"
2023-01-09T04:53:08.425843840Z I0109 04:53:08.425714       1 tlsconfig.go:178] "Loaded client CA" index=0 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"admin-kubeconfig-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:21 +0000 UTC to 2033-01-06 04:28:21 +0000 UTC (now=2023-01-09 04:53:08.425694159 +0000 UTC))"
2023-01-09T04:53:08.425843840Z I0109 04:53:08.425743       1 tlsconfig.go:178] "Loaded client CA" index=1 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"kubelet-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:24 +0000 UTC to 2023-01-10 04:28:24 +0000 UTC (now=2023-01-09 04:53:08.425728116 +0000 UTC))"
2023-01-09T04:53:08.425843840Z I0109 04:53:08.425760       1 shared_informer.go:262] Caches are synced for RequestHeaderAuthRequestController
2023-01-09T04:53:08.425843840Z I0109 04:53:08.425781       1 tlsconfig.go:178] "Loaded client CA" index=2 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"kube-control-plane-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:24 +0000 UTC to 2024-01-09 04:28:24 +0000 UTC (now=2023-01-09 04:53:08.425751813 +0000 UTC))"
2023-01-09T04:53:08.425843840Z I0109 04:53:08.425831       1 tlsconfig.go:178] "Loaded client CA" index=3 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"kube-apiserver-to-kubelet-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:24 +0000 UTC to 2024-01-09 04:28:24 +0000 UTC (now=2023-01-09 04:53:08.425792991 +0000 UTC))"
2023-01-09T04:53:08.425893228Z I0109 04:53:08.425859       1 tlsconfig.go:178] "Loaded client CA" index=4 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"kubelet-bootstrap-kubeconfig-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:22 +0000 UTC to 2033-01-06 04:28:22 +0000 UTC (now=2023-01-09 04:53:08.425843663 +0000 UTC))"
2023-01-09T04:53:08.425903819Z I0109 04:53:08.425897       1 tlsconfig.go:178] "Loaded client CA" index=5 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"kube-csr-signer_@1673239265\" [] issuer=\"kubelet-signer\" (2023-01-09 04:41:04 +0000 UTC to 2023-01-10 04:28:24 +0000 UTC (now=2023-01-09 04:53:08.425868162 +0000 UTC))"
2023-01-09T04:53:08.425941162Z I0109 04:53:08.425923       1 tlsconfig.go:178] "Loaded client CA" index=6 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"openshift-kube-apiserver-operator_node-system-admin-signer@1673239266\" [] issuer=\"<self>\" (2023-01-09 04:41:06 +0000 UTC to 2024-01-09 04:41:07 +0000 UTC (now=2023-01-09 04:53:08.425907012 +0000 UTC))"
2023-01-09T04:53:08.425954472Z I0109 04:53:08.425946       1 tlsconfig.go:178] "Loaded client CA" index=7 certName="client-ca::kube-system::extension-apiserver-authentication::client-ca-file,client-ca::kube-system::extension-apiserver-authentication::requestheader-client-ca-file" certDetail="\"aggregator-signer\" [] issuer=\"<self>\" (2023-01-09 04:28:22 +0000 UTC to 2023-01-10 04:28:22 +0000 UTC (now=2023-01-09 04:53:08.425936072 +0000 UTC))"
2023-01-09T04:53:08.426080289Z I0109 04:53:08.426066       1 tlsconfig.go:200] "Loaded serving cert" certName="serving-cert::/var/run/secrets/serving-cert/tls.crt::/var/run/secrets/serving-cert/tls.key" certDetail="\"api.openshift-oauth-apiserver.svc\" [serving] validServingFor=[api.openshift-oauth-apiserver.svc,api.openshift-oauth-apiserver.svc.cluster.local] issuer=\"openshift-service-serving-signer@1673239264\" (2023-01-09 04:41:24 +0000 UTC to 2025-01-08 04:41:25 +0000 UTC (now=2023-01-09 04:53:08.426048395 +0000 UTC))"
2023-01-09T04:53:08.426176974Z I0109 04:53:08.426164       1 named_certificates.go:53] "Loaded SNI cert" index=0 certName="self-signed loopback" certDetail="\"apiserver-loopback-client@1673239987\" [serving] validServingFor=[apiserver-loopback-client] issuer=\"apiserver-loopback-client-ca@1673239987\" (2023-01-09 03:53:07 +0000 UTC to 2024-01-09 03:53:07 +0000 UTC (now=2023-01-09 04:53:08.42615186 +0000 UTC))"
2023-01-09T04:53:57.596156278Z W0109 04:53:57.595206       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.596934405Z W0109 04:53:57.596676       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.596934405Z W0109 04:53:57.596721       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.596934405Z W0109 04:53:57.596726       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.596934405Z W0109 04:53:57.596897       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.597067373Z W0109 04:53:57.597000       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.597186488Z W0109 04:53:57.597103       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.597330535Z W0109 04:53:57.597304       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:57.599067434Z W0109 04:53:57.599035       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.595548752Z W0109 04:53:58.595514       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.597757278Z W0109 04:53:58.597727       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.597832560Z W0109 04:53:58.597789       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.597832560Z W0109 04:53:58.597791       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.597832560Z W0109 04:53:58.597791       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.597849837Z W0109 04:53:58.597828       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.597857905Z W0109 04:53:58.597831       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.597916100Z W0109 04:53:58.597898       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:58.600272873Z W0109 04:53:58.600236       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:53:59.904594778Z W0109 04:53:59.904556       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.001331514Z W0109 04:54:00.001291       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.140951312Z W0109 04:54:00.140914       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.246305518Z W0109 04:54:00.246262       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.254466119Z W0109 04:54:00.254444       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.267719789Z W0109 04:54:00.267695       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.311102837Z W0109 04:54:00.311072       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.445773282Z W0109 04:54:00.445730       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:00.450000456Z W0109 04:54:00.449978       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.210766095Z W0109 04:54:02.210725       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.350407201Z W0109 04:54:02.350364       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.566260563Z W0109 04:54:02.566212       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.618999012Z W0109 04:54:02.618953       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.645351420Z W0109 04:54:02.645314       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.697706716Z W0109 04:54:02.697663       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.825638298Z W0109 04:54:02.825599       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:02.911009546Z W0109 04:54:02.910971       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:03.517067867Z W0109 04:54:03.517032       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:05.491487947Z W0109 04:54:05.491443       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:06.352583434Z W0109 04:54:06.352545       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:06.365017567Z W0109 04:54:06.364996       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:06.377301894Z W0109 04:54:06.377273       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:06.519118833Z W0109 04:54:06.519081       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:06.819963958Z W0109 04:54:06.819923       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:06.980523846Z W0109 04:54:06.980487       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:07.016027597Z W0109 04:54:07.015988       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:08.309334396Z W0109 04:54:08.309297       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:11.490955272Z W0109 04:54:11.490917       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:12.679472422Z W0109 04:54:12.679428       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:12.991903301Z W0109 04:54:12.991859       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:13.412935908Z W0109 04:54:13.412890       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:13.515146703Z W0109 04:54:13.515111       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:13.585737793Z W0109 04:54:13.585704       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:14.028014279Z W0109 04:54:14.027973       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:14.188526909Z W0109 04:54:14.188484       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:14.705041956Z W0109 04:54:14.704996       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:20.607164917Z W0109 04:54:20.607123       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:21.608280337Z W0109 04:54:21.608242       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:22.238634958Z W0109 04:54:22.238595       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:22.709637612Z W0109 04:54:22.709596       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:22.944249793Z W0109 04:54:22.944214       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:23.083714123Z W0109 04:54:23.083678       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:23.173073973Z W0109 04:54:23.173039       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:23.381557478Z W0109 04:54:23.381525       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:23.637294799Z W0109 04:54:23.637257       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:24.162741565Z W0109 04:54:24.162699       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:24.309144093Z W0109 04:54:24.309094       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:25.046788865Z W0109 04:54:25.046749       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:25.148447753Z W0109 04:54:25.148404       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:29.280261951Z W0109 04:54:29.280214       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:36.026648692Z W0109 04:54:36.026604       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:36.767307832Z W0109 04:54:36.767269       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:38.122655303Z W0109 04:54:38.122618       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:38.734205451Z W0109 04:54:38.734165       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:40.392741702Z W0109 04:54:40.392696       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:40.553679047Z W0109 04:54:40.553641       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:43.022435406Z W0109 04:54:43.022396       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:43.219107143Z W0109 04:54:43.219071       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:43.300429204Z W0109 04:54:43.300391       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:54:43.988878089Z W0109 04:54:43.988844       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.652392435Z W0109 04:56:10.652325       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.664994358Z W0109 04:56:10.664964       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.666244910Z W0109 04:56:10.666216       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.666279090Z W0109 04:56:10.666263       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.666311301Z W0109 04:56:10.666296       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.666368363Z W0109 04:56:10.666347       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.666514581Z W0109 04:56:10.666481       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.666600206Z W0109 04:56:10.666493       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:10.666600206Z W0109 04:56:10.666494       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.654131518Z W0109 04:56:11.654099       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.667454775Z W0109 04:56:11.667416       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.667556901Z W0109 04:56:11.667465       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.667556901Z W0109 04:56:11.667507       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.667609752Z W0109 04:56:11.667594       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.667620240Z W0109 04:56:11.667604       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.668545342Z W0109 04:56:11.668525       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.668566068Z W0109 04:56:11.668529       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:11.668566068Z W0109 04:56:11.668559       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:12.969827621Z W0109 04:56:12.969768       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.108018516Z W0109 04:56:13.107983       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.129249879Z W0109 04:56:13.129220       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.197672874Z W0109 04:56:13.197635       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.272127750Z W0109 04:56:13.272096       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.382972161Z W0109 04:56:13.382926       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.413247050Z W0109 04:56:13.413211       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.440543784Z W0109 04:56:13.440503       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:13.522159230Z W0109 04:56:13.522125       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.273103204Z W0109 04:56:15.273067       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.494221662Z W0109 04:56:15.494187       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.572643743Z W0109 04:56:15.572609       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.647172218Z W0109 04:56:15.647140       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.666443690Z W0109 04:56:15.666410       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.782771057Z W0109 04:56:15.782732       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:15.908595775Z W0109 04:56:15.908558       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:16.057838084Z W0109 04:56:16.057789       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:16.145468527Z W0109 04:56:16.145437       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:19.512249859Z W0109 04:56:19.512213       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:19.716979013Z W0109 04:56:19.716943       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:19.770352820Z W0109 04:56:19.770319       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:19.969269071Z W0109 04:56:19.969234       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:19.988830944Z W0109 04:56:19.988781       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:20.017134133Z W0109 04:56:20.017102       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:20.031405290Z W0109 04:56:20.031376       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:20.081592624Z W0109 04:56:20.081556       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:20.850010726Z W0109 04:56:20.849968       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:25.150530472Z W0109 04:56:25.150482       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:25.548105421Z W0109 04:56:25.548066       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:26.330749265Z W0109 04:56:26.330705       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:26.408143965Z W0109 04:56:26.408080       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:27.055797489Z W0109 04:56:27.055760       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:27.282142902Z W0109 04:56:27.282103       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:27.313530187Z W0109 04:56:27.313491       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:27.393210530Z W0109 04:56:27.393165       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:27.878245431Z W0109 04:56:27.878197       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:29.720541582Z W0109 04:56:29.720500       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:30.722540200Z W0109 04:56:30.722501       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:32.562169672Z W0109 04:56:32.562121       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:35.131063206Z W0109 04:56:35.131024       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:35.135310404Z W0109 04:56:35.135285       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:35.825690517Z W0109 04:56:35.825650       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:35.932082074Z W0109 04:56:35.932044       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:36.427362366Z W0109 04:56:36.427327       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:36.576351689Z W0109 04:56:36.576308       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:37.928321456Z W0109 04:56:37.928275       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:38.919066929Z W0109 04:56:38.919028       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:39.173260713Z W0109 04:56:39.173222       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:39.424223358Z W0109 04:56:39.424183       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:39.655297563Z W0109 04:56:39.655259       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:45.187971864Z W0109 04:56:45.187927       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:50.291373294Z W0109 04:56:50.291330       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:50.690029749Z W0109 04:56:50.689997       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:52.007190171Z W0109 04:56:52.007152       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:52.880177217Z W0109 04:56:52.880140       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:53.142334097Z W0109 04:56:53.142294       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:53.457019397Z W0109 04:56:53.456974       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:54.158864761Z W0109 04:56:54.158827       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:54.944418087Z W0109 04:56:54.944380       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:56:55.604367916Z W0109 04:56:55.604327       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.199.219:2379 10.0.199.219 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.199.219:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.144931248Z W0109 04:58:03.144896       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.145166143Z W0109 04:58:03.145043       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.145166143Z W0109 04:58:03.145080       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.151465577Z W0109 04:58:03.150987       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.152374142Z W0109 04:58:03.152330       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.152425641Z W0109 04:58:03.152404       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.153225561Z W0109 04:58:03.153200       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.157023819Z W0109 04:58:03.157002       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:03.157337874Z W0109 04:58:03.157319       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.146788345Z W0109 04:58:04.146745       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.146911724Z W0109 04:58:04.146747       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.146958046Z W0109 04:58:04.146789       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.152970023Z W0109 04:58:04.152937       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.156652710Z W0109 04:58:04.154012       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.156652710Z W0109 04:58:04.154075       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.156652710Z W0109 04:58:04.154116       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.158470348Z W0109 04:58:04.158442       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:04.158714729Z W0109 04:58:04.158473       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.439030809Z W0109 04:58:05.438987       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.509414481Z W0109 04:58:05.509373       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.521453943Z W0109 04:58:05.521425       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.618689920Z W0109 04:58:05.618655       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.784148019Z W0109 04:58:05.784108       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.810125947Z W0109 04:58:05.810086       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.816000490Z W0109 04:58:05.815971       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.900126142Z W0109 04:58:05.900089       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:05.918325271Z W0109 04:58:05.918292       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:07.733214283Z W0109 04:58:07.733174       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:07.743156358Z W0109 04:58:07.743129       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:07.902678649Z W0109 04:58:07.902640       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.010865284Z W0109 04:58:08.010817       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.263339110Z W0109 04:58:08.263302       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.541558873Z W0109 04:58:08.541524       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.554437548Z W0109 04:58:08.554408       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.555193580Z W0109 04:58:08.555178       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.656964641Z W0109 04:58:08.656931       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.734965867Z W0109 04:58:08.734935       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:08.847015609Z W0109 04:58:08.846984       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:10.274286698Z W0109 04:58:10.274246       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:11.431748044Z W0109 04:58:11.431716       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:11.808397699Z W0109 04:58:11.808363       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:12.216422757Z W0109 04:58:12.216384       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:12.255418157Z W0109 04:58:12.255380       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:12.405884465Z W0109 04:58:12.405847       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:12.710136258Z W0109 04:58:12.710103       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:12.931584081Z W0109 04:58:12.931546       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:13.252957681Z W0109 04:58:13.252916       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:13.293952543Z W0109 04:58:13.293910       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:13.345951462Z W0109 04:58:13.345920       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:17.201873134Z W0109 04:58:17.201833       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:18.345183745Z W0109 04:58:18.345144       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:18.748472854Z W0109 04:58:18.748438       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:18.961816470Z W0109 04:58:18.961770       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:19.177651939Z W0109 04:58:19.177613       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:19.553744299Z W0109 04:58:19.553699       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:19.660946070Z W0109 04:58:19.660909       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:20.336738563Z W0109 04:58:20.336697       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:20.789997098Z W0109 04:58:20.789958       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:20.921131836Z W0109 04:58:20.921097       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:23.878731794Z W0109 04:58:23.878688       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:27.022139927Z W0109 04:58:27.022102       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:28.795753667Z W0109 04:58:28.795716       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:29.132020385Z W0109 04:58:29.131974       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:30.085490558Z W0109 04:58:30.085446       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:30.177377197Z W0109 04:58:30.177337       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:30.203541040Z W0109 04:58:30.203499       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:30.296474050Z W0109 04:58:30.296439       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:31.487760698Z W0109 04:58:31.487725       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:32.204991302Z W0109 04:58:32.204952       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:40.558689296Z W0109 04:58:40.558649       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:46.131261725Z W0109 04:58:46.131219       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:46.184377550Z W0109 04:58:46.184337       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:46.977909667Z W0109 04:58:46.977859       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:48.008077107Z W0109 04:58:48.008030       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:48.914878883Z W0109 04:58:48.914832       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:49.079480410Z W0109 04:58:49.079441       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:50.197263044Z W0109 04:58:50.197223       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:58:50.337660728Z W0109 04:58:50.337616       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:59:03.254862341Z W0109 04:59:03.254823       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:59:05.638883793Z W0109 04:59:05.638830       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T04:59:06.640004115Z W0109 04:59:06.639969       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.160.211:2379 10.0.160.211 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.160.211:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.678787384Z W0109 05:00:01.678748       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.679068751Z W0109 05:00:01.679044       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.679264592Z W0109 05:00:01.679199       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.679365283Z W0109 05:00:01.679348       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.679444943Z W0109 05:00:01.679430       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.679588912Z W0109 05:00:01.679557       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.679694346Z W0109 05:00:01.679587       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.679737273Z W0109 05:00:01.679711       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:01.679786629Z W0109 05:00:01.679624       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.680128210Z W0109 05:00:02.680091       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.680184099Z W0109 05:00:02.680161       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.680184099Z W0109 05:00:02.680090       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.680205817Z W0109 05:00:02.680092       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.680214444Z W0109 05:00:02.680090       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.680236298Z W0109 05:00:02.680106       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.680246875Z W0109 05:00:02.680118       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.680256344Z W0109 05:00:02.680133       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:02.680263929Z W0109 05:00:02.680139       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.078850084Z W0109 05:00:04.078816       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.220311817Z W0109 05:00:04.220270       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.261663991Z W0109 05:00:04.261630       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.343159101Z W0109 05:00:04.343127       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.375693859Z W0109 05:00:04.375661       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.428004274Z W0109 05:00:04.427963       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.552634615Z W0109 05:00:04.552593       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.557930187Z W0109 05:00:04.557900       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:04.566224028Z W0109 05:00:04.566197       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:06.224327752Z W0109 05:00:06.224287       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:06.516211153Z W0109 05:00:06.516172       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:06.540780175Z W0109 05:00:06.540741       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:06.662423233Z W0109 05:00:06.662391       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:06.891059769Z W0109 05:00:06.891020       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:07.010583462Z W0109 05:00:07.010542       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:07.118219590Z W0109 05:00:07.118184       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:07.278508253Z W0109 05:00:07.278468       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:07.384284133Z W0109 05:00:07.384237       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:09.970029372Z W0109 05:00:09.969990       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:10.017416282Z W0109 05:00:10.017381       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:10.570952700Z W0109 05:00:10.570915       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:10.771187952Z W0109 05:00:10.771145       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:10.785514753Z W0109 05:00:10.785486       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.069557416Z W0109 05:00:11.069521       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.083900656Z W0109 05:00:11.083872       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.494388485Z W0109 05:00:11.494356       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.786299394Z W0109 05:00:11.786259       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.819638871Z W0109 05:00:11.819608       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:11.954249806Z W0109 05:00:11.954213       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:13.569544787Z W0109 05:00:13.569504       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:15.929235988Z W0109 05:00:15.929197       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:16.695377262Z W0109 05:00:16.695340       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:16.775711272Z W0109 05:00:16.775675       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:16.816021612Z W0109 05:00:16.815988       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:17.699158573Z W0109 05:00:17.699121       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:17.716478341Z W0109 05:00:17.716447       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:18.010593790Z W0109 05:00:18.010557       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:18.019878548Z W0109 05:00:18.019836       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:18.656660347Z W0109 05:00:18.656620       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:18.878061585Z W0109 05:00:18.878026       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:20.493720672Z W0109 05:00:20.493676       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:26.613977106Z W0109 05:00:26.613939       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:26.868540668Z W0109 05:00:26.868500       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:27.162833669Z W0109 05:00:27.162785       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:27.390706661Z W0109 05:00:27.390668       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:27.420530468Z W0109 05:00:27.420488       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:27.643894222Z W0109 05:00:27.643851       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:27.952954807Z W0109 05:00:27.952920       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:28.673462677Z W0109 05:00:28.673427       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:29.402166369Z W0109 05:00:29.402122       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:31.140605628Z W0109 05:00:31.140567       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:41.008082600Z W0109 05:00:41.008036       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:41.433744226Z W0109 05:00:41.433698       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:42.465150164Z W0109 05:00:42.465115       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:42.544534317Z W0109 05:00:42.544493       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:43.869106666Z W0109 05:00:43.869064       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:44.607900271Z W0109 05:00:44.607829       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:45.455297560Z W0109 05:00:45.455253       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:46.486412040Z W0109 05:00:46.486366       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
2023-01-09T05:00:48.788276077Z W0109 05:00:48.788237       1 clientconn.go:1331] [core] grpc: addrConn.createTransport failed to connect to {10.0.145.4:2379 10.0.145.4 <nil> 0 <nil>}. Err: connection error: desc = "transport: Error while dialing dial tcp 10.0.145.4:2379: connect: connection refused". Reconnecting...
